agent/sources/client/config/agent.conf

```diff
@@ -0,0 +1,26 @@
+# This file configures the way the Rudder agent connects to its server
+
+server = "rudder.example.com"
```

Comment:
Server default value should be read from policy_server.dat

Reply:
if we distribute this file as a default configuration file, we don't want to have any value here

Reply:
indeed